framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -742,4 +735,17 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 		// Update logs
 		this.logger.info({ transactionID: result.transactionId }, 'Sent CCU transaction');
 	}
+
+	private async _initializeReceivingChainClient() {
+		if (!this.config.receivingChainIPCPath && !this.config.receivingChainWsURL) {
+			throw new Error('IPC path and WS url are undefined.');
+		}
+		if (this.config.receivingChainIPCPath) {
+			this._receivingChainClient = await apiClient.createIPCClient(
+				this.config.receivingChainIPCPath,
+			);
+		} else if (this.config.receivingChainWsURL) {
```

Comment:
This can just be `else` instead of `else if`

Reply:
else if to check if `this.config.receivingChainWsURL` is not undefined

Reply:
I think it is already checked on line 740. And if it does not pass line 743 then it must mean `this.config.receivingChainWsURL` exist

Reply:
unfortunately lint throws error that `this.config.receivingChainWsURL` can be undefined 😢